### PR TITLE
Open icon editor panel on double clicking an icon

### LINKED ISSUE #118: Feature Request: Open Icon editor panel on double clicking an icon
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Icons just get selected and deselected when a user double clicks on them. User should get icon editor panel on double clicking an icon.

### Expected Behavior

User should get the icon editor panel on double clicking fastly on an icon.

### Steps To Reproduce

1. Double click on any icon
2. Icon gets selected and deselected on fast as well as slow double clicks

### Anything else?

_No response_

### What browsers are you seeing the problem on?

_No response_

### PR DESCRIPTION
**Pull request by ayushsatyam146**
Fixes #118. Enabling functionality for opening icon editor panel on double clicking an icon. Attaching a video showing current behavior. 

https://user-images.githubusercontent.com/55888723/159130982-f50e03a9-5c4e-4fa3-b51b-fb478ddb9dd1.mp4

### CODE REVIEW

File: src/modules/IconsSet.js [hunk 1]
```diff
@@ -566,17 +574,20 @@ const IconsSet = (props) => {
...
+                      setSearchValue(icon)
+                      if (selectMultiple) {
+                        window.history.replaceState(
+                          '',
+                          'EOS Icons',
+                          `${window.location.pathname}?iconName=${icon}&type=animated`
+                        )
+                      }
                     }
+                    if (event.detail === 2) iconEditorToggle()
```
Comment (Abhirup-99) [inline src/modules/IconsSet.js:590]: Can we move this to a function?

**Comment by cyntss** [comment]
I really like this feature! I've tested it and it works fine. We can add UX improvements later on in new PRs

File: src/modules/IconsSet.js (hunk 1 above)
Comment (cyntss) [reply src/modules/IconsSet.js:590]: this is a good catch @Abhirup-99 !

**Review by Abhirup-99 — APPROVED** [review]

**Comment by ayushsatyam146** [comment]
@Abhirup-99 resolved the conflicts of this branch and made the required changes. Please review and merge

**Review by Abhirup-99 — APPROVED** [review]

**Pull request merged**